src/components/KCheckbox/KCheckbox.vue

```diff
@@ -59,4 +84,63 @@ export default defineComponent({
 @import '@/styles/variables';
 @import '@/styles/functions';
 
+.k-checkbox-label {
+  /* Body/Popover Content - Regular - 400 */
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  /* identical to box height, or 143% */
+  display: inline-block;
+  align-items: center;
+
+  /* Black/85 */
+  color: var(--black-85, rgba(0, 0, 0, 0.85));
+
+  /* Inside auto layout */
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.k-checkbox-description {
+  padding-top: var(--spacing-xxs);
+  padding-left: var(--spacing-lg);
+  /* Body/Popover Content - Regular - 400 */
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  /* identical to box height, or 143% */
+
+  /* Black/45 */
+  color: var(--black-45, rgba(0, 0, 0, 0.45));
+
+  /* Inside auto layout */
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.k-checkbox-label:has(+ .k-checkbox-description) {
+  /* Headline/H4 - Semibold - 600
+  .type-sm
+  */
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14px;
```

Comment:
nit: same comment for all instances of font-size
```suggestion
  font-size: var(--type-sm, type(sm));
```